### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Kaiserabbas**
This branch was created to add a dropdown mobile menu in the responsive portfolio portal. I have added the following to the mobile nav bar with the help of my coding partner. 
- added a new dropdown navbar.
- added functionality by Javascript code.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @Kaiserabbas , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project that does not  have all the required linters setup. Please setup [all required linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kaiserabbas** [comment]
I have added all linters but the "Webhint error" issue is arising there for a few weeks.

File: index.html [hunk 1]
```diff
@@ -6,21 +6,28 @@
...
+          <button class="menu-button" onclick="toggleDropdown()"> <i class="fa fa-bars"></i></button> 
         </div>
+        <div class="dropdown">
+          <div id="dropdownContent" class="dropdown-content">
+          <span class="close" onclick="closeDropdown()">&times;</span>
+          <a href="./index.html" onclick="closeDropdown()">Portfolio</a>
+          <a href="#about-me" onclick="closeDropdown()">About</a>
+          <a href="#contact-me" onclick="closeDropdown()">Contact</a>
+        </div>
+    </div>
```
Comment (mckent05) [inline index.html:26]: - Good job done so far, but you have not styled your mobile menu according to the Figma template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
ethaehre

Design template:
ryjsrtjsetj

**OPTIONAL** - Please, it is not advisable to call Javascript functions in your HTML file. Ensure all DOM manipulations are done in the js file. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating the mobile menu for your project 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kaiserabbas) [reply index.html:26]: I have done the changes required to match figma design has been completed. Kindly approve.

Comment (wayungi) [inline index.html:26]: - [ ] Please note that the Figma has mobilenavigation has a blur. Please[ add a blur](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) to your mobile navigation in order to match the figma.  👏🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kaiserabbas** [comment]
Background blur is done. 
Kindly approve.
image

**Review by cynthiainga — APPROVED** [review]
Hi @Kaiserabbas, 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Good design ✔️ 
- [ ] The menu works ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**